src/auto/enums.rs

```diff
@@ -40,3 +40,181 @@ impl FromGlib<ffi::PangoAlignment> for Alignment {
     }
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
+pub enum Stretch {
+    UltraCondensed,
+    ExtraCondensed,
+    Condensed,
+    SemiCondensed,
+    Normal,
+    SemiExpanded,
+    Expanded,
+    ExtraExpanded,
+    UltraExpanded,
+    #[doc(hidden)]
+    __Nonexhaustive(()),
+}
+
+#[doc(hidden)]
+impl ToGlib for Stretch {
```

Comment:
I don't see removing `pub use ffi::PangoStretch as Stretch;` from `lib.rs` for this and others.
And this line override `pub use auto::*` and generating will be unusable to enduser.
Can you generate all reexported and remove reexports.